src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    public static final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
```suggestion
        return Color.values()[index].name();
```
from common [mistakes](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md):
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/338a7c3a-192d-4619-ae76-4b866d59dc9d)

Reply:
Probably last requested change, don't give up, you're almost there  :) 💪